backend/backend.go

```diff
@@ -186,6 +186,20 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 	backend.ratesUpdater = rates.NewRateUpdater(hclient)
 	backend.ratesUpdater.Observe(backend.Notify)
 	backend.ratesUpdater.Start()
+	for _, fiat := range config.AppConfig().Backend.FiatList {
+		if config.AppConfig().Backend.BitcoinActive {
```

Comment:
maybe using `CoinActive()` is better, like here? https://github.com/digitalbitbox/bitbox-wallet-app/blob/eeff1391b11b658e07bfb5a843b821c15cb79892/backend/backend.go#L337